packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -234,7 +239,7 @@ async function moveTeam({
   await addTeamRedirect({
     oldTeamSlug: team.slug,
     teamSlug: newSlug,
-    orgSlug: org.slug,
+    orgSlug: org.slug || (orgMetadata?.requestedSlug ?? null),
```

Comment:
Found this while doing test that in case payment is required for the organization(which is true for cal.com), org.slug won't be set and thus we need to fallback to requestedSlug